integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java

```diff
@@ -335,16 +335,17 @@ private int readdirInternal(String path, long buff, long filter, long offset,
 
   @Override
   public int open(String path, FuseFileInfo fi) {
-    return AlluxioFuseUtils.call(LOG, () -> openInternal(path, fi), "Fuse.Open", "path=%s", path);
+    final int flags = fi.flags.get();
+    boolean overwrite = OpenFlags.valueOf(flags) == OpenFlags.O_WRONLY;
+    String methodName = overwrite ? "Fuse.OpenOverwrite" : "Fuse.Open";
```

Comment:
we can tell if this is `Fuse.OpenOverwrite` from the flag you are logging now, right?
Not sure if it more helpful to differentiate "Fuse.OpenOverwrite" and "Fuse.Open".

Reply:
Logging and metrics are two systems. From logging, we can know it's open or openOverwrite, but from metrics, the previous method name cannot differentiate these two circumstances.